src/intellisense/ffi.rs

```diff
@@ -0,0 +1,753 @@
+        const OMPTaskwaitDirective = 245;  // OpenMP taskwait directive.
+        const OMPFlushDirective = 246;  // OpenMP flush directive.
+        const SEHLeaveStmt = 247;  // Windows Structured Exception Handling's leave statement.
+        const OMPOrderedDirective = 248;  // OpenMP ordered directive.
+        const OMPAtomicDirective = 249;  // OpenMP atomic directive.
+        const OMPForSimdDirective = 250;  // OpenMP for SIMD directive.
+        const OMPParallelForSimdDirective = 251;  // OpenMP parallel for SIMD directive.
+        const OMPTargetDirective = 252;  // OpenMP target directive.
+        const OMPTeamsDirective = 253;  // OpenMP teams directive.
+        const OMPTaskgroupDirective = 254;  // OpenMP taskgroup directive.
+        const OMPCancellationPointDirective = 255;  // OpenMP cancellation point directive.
+        const OMPCancelDirective = 256;  // OpenMP cancel directive.
+        const LastStmt = DxcCursorKind::OMPCancelDirective.bits;
+
+        const TranslationUnit = 300; // Cursor that represents the translation unit itself.
+
+        /* Attributes */
+        const FirstAttr = 400;
+        /**
+         * \brief An attribute whose specific kind is not exposed via this
+        * interface.
+        */
+        const UnexposedAttr = 400;
+
+        const IBActionAttr = 401;
+        const IBOutletAttr = 402;
+        const IBOutletCollectionAttr = 403;
+        const CXXFinalAttr = 404;
+        const CXXOverrideAttr = 405;
+        const AnnotateAttr = 406;
+        const AsmLabelAttr = 407;
+        const PackedAttr = 408;
+        const PureAttr = 409;
+        const ConstAttr = 410;
+        const NoDuplicateAttr = 411;
+        const CUDAConstantAttr = 412;
+        const CUDADeviceAttr = 413;
+        const CUDAGlobalAttr = 414;
+        const CUDAHostAttr = 415;
+        const CUDASharedAttr = 416;
+        const LastAttr = DxcCursorKind::CUDASharedAttr.bits;
+
+        /* Preprocessing */
+        const PreprocessingDirective = 500;
+        const MacroDefinition = 501;
+        const MacroExpansion = 502;
+        const MacroInstantiation = DxcCursorKind::MacroExpansion.bits;
+        const InclusionDirective = 503;
+        const FirstPreprocessing = DxcCursorKind::PreprocessingDirective.bits;
+        const LastPreprocessing = DxcCursorKind::InclusionDirective.bits;
+
+        /* Extra Declarations */
+        /**
+         * \brief A module import declaration.
+        */
+        const ModuleImportDecl = 600;
+        const FirstExtraDecl = DxcCursorKind::ModuleImportDecl.bits;
+        const LastExtraDecl = DxcCursorKind::ModuleImportDecl.bits;
+    }
+}
+
+iid!(pub IID_IDxcDiagnostic = 0x4f76b234, 0x3659, 0x4d33, 0x99, 0xb0, 0x3b, 0x0d, 0xb9, 0x94, 0xb5, 0x64);
+com_interface! {
+    interface IDxcDiagnostic: IUnknown{
+        iid: IID_IDxcDiagnostic,
+        vtable: IDxcDiagnosticVtbl,
+
+        fn FormatDiagnostic(options: DxcDiagnosticDisplayOptions, result: *mut LPSTR) -> HRESULT;
```

Comment:
For this file, I think we should make sure that the capitalization is consistent with the rust standard (and the rest of the code base).

Reply:
Changed file's methods from the camel case on the reference dxcisense.h to rust's snake case